src/feature-gates.md

```diff
@@ -27,6 +27,13 @@ For example:
 (active, non_ascii_idents, "1.0.0", Some(55467), None),
 ```
 
+Features can be marked as incomplete, and trigger the warn-by-default [`incomplete_features` lint](https://doc.rust-lang.org/rustc/lints/listing/warn-by-default.html#incomplete-features) by setting their type to `incomplete`:
```

Comment:
Please use a reference-style link, i.e.

```suggestion
Features can be marked as incomplete, and trigger the warn-by-default [`incomplete_features` lint] by setting their type to `incomplete`:
```

And at the end of the section or chapter:
```md
[`incomplete_features` lint]: https://doc.rust-lang.org/rustc/lints/listing/warn-by-default.html#incomplete-features
```